### DISCUSSION TITLE
When will primevue templates (Freya for example) use tailwindcss instead of primeflex?

### DISCUSSION THREAD

**Discussion by sergeydus** [primefaces/community#1615] [PrimeVue Templates] [answered]
recently bought the Freya template, am curious when will it be converted to use tailwindcss instead

**Comment by cantin-f** [comment]
Never got answer to this question here : https://github.com/orgs/primefaces/discussions/1174

**Reply by sergeydus** [reply]
that's unfortunate ;(
glad I'm not the only one asking this question though, might get them to work on it

**Comment by tugcekucukoglu** [comment, marked as answer]
The new version of Freya with Tailwind can be downloaded from the PrimeStore.
